packages/design-system/src/components/reset/resetLists.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+
+export default {
+  title: 'Reset / Lists',
+};
+
+export const Lists = () => {
+  return (
+    <article>
+      <hgroup>
+        <h2>Lists</h2>
+        <p>This page showcases how reset styles impact HTML list tags.</p>
+      </hgroup>
+
+      <article>
+        <h3>Ordered lists</h3>
+
+        <div className="preview">
+          <ol>
+            <li>List Item 1</li>
+            <li>List Item 2</li>
+            <li>List Item 3</li>
+            <li>List Item 4</li>
+          </ol>
+
+          <hr />
+
+          <ol>
+            <li>List Item 1</li>
+            <li>List Item 2</li>
+            <li>
+              List Item 3
+              <ol>
```

Comment:
I need design 👀 on nested lists. We don't have examples and the list marker is the browser default currently. Also need to figure out how spacing works for nested `ul`/`ol` since they have `margin-top` applied

Reply:
I'd be happy to work on this with you @zarahzachz

Reply:
I worked with Scott last week and we came up with an ideal pattern for ordered lists. Updated the Story to reflect this decision.